php/Terminus/Models/Environment.php

```diff
@@ -140,6 +162,36 @@ public function cloneFiles($to_env) {
   }
 
   /**
+   * Commits changes to code
+   *
+   * @param [string] $commit Should be the commit message to use if committing
+   *   on server changes
+   * @return [array] $data['data']
+   */
+  public function commitChanges($commit = null) {
+    ob_start();
+    passthru('git config --global user.email');
+    $git_email = ob_get_clean();
+    ob_start();
+    passthru('git config --global user.name');
+    $git_user = ob_get_clean();
```

Comment:
Doing it with system(string) makes it output to terminal when run. Using the object buffer stops that.